cores/esp32/Esp.cpp

```diff
@@ -226,6 +226,8 @@ String EspClass::getSketchMD5()
         md5.add(buf.get(), readBytes);
         lengthLeft -= readBytes;
         offset += readBytes;
+
```

Comment:
You can use the following to delay only when single core (also applies to ESP32-S2 and ESP32-C3)
```cpp
#if CONFIG_FREERTOS_UNICORE
delay(1);  // Fix solo WDT
#endif
```